package.json

```diff
@@ -19,7 +19,8 @@
     "react-redux": "^8.1.2",
     "react-router-dom": "^6.15.0",
     "react-scripts": "5.0.1",
-    "typescript": "^4.9.5",
+    "swiper": "^10.2.0",
+    "typescript": "^5.2.2",
```

Comment:
Is it ok that the typescript version changed?

Reply:
Yep. Otherwise, Swipe components won't work.

Reply:
Oooh, cool. Good to know)